### PR TITLE
Add UMA tutorial

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
- Add user managed access tutorial
- Add instructions for deploying photo editor and photo viewer

### CODE REVIEW

File: en/docs/learn/configuring-access-delegation-with-uma.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Configuring Access Delegation with OAuth 2.0
+
+User Managed Access (UMA) enables controlled access to a user's protected digital resources using a centralized authorization server with authorization policies. This tutorial demonstrates giving access to a user's photo resources in a sample application by configuring UMA 2.0 in WSO2 Identity Server (WSO2 IS).
+
+## Scenario 
+
+Lily, Oliver and Pam are three users in WSO2 IS. Lily has a photo album in the **Photo-editer** application and wants to share the Family photos album with her family members; Oliver and Pam . Oliver and Pam can view this photo album using the **Photo-viewer** application. The **Photo-editor** and **Photo-viewer** applications are using WSO2 IS as their identity provider.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-uma.md]: Oliver,

Comment (YvonneW) [reply en/docs/learn/configuring-access-delegation-with-uma.md]: Oliver and Pam.

File: en/docs/learn/configuring-access-delegation-with-uma.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ```
+
+    ``` java tab="Windows"
+    wso2server.bat run
+    ```
+	
+3. Follow the steps in
+   [deploying the **photo-editor** and **photo-view** webapps](../../learn/deploying-the-sample-app/#deploying-the-photo-editor-and-photo-viewer-webapps) to download, deploy, and register **Photo Editor** sample.
+
+Once you have deployed the samples, note that two service providers have been created on the [management console](https://localhost:9443/carbon) using dynamic client registration. You can access or edit them through the management console.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-uma.md]: Management Console

File: en/docs/learn/configuring-access-delegation-with-uma.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Once you have deployed the samples, note that two service providers have been created on the [management console](https://localhost:9443/carbon) using dynamic client registration. You can access or edit them through the management console. 
+    ![photo-samples-service-providers](../assets/img/learn/photo-samples-service-providers.png)
+
+## Create users
+
+1. Log in using admin/admin credentials and create a new user called "Lily" (the resource owner) with login permission. For instructions, see [Adding Users and Roles](../../learn/adding-users-and-roles#create-a-user).
+
+    !!! info
+        `admin` is the default administrative user in WSO2 Identity Server.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-uma.md]: admin

File: en/docs/learn/configuring-access-delegation-with-uma.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+2. Log in to the **Photo Editor** application using Lily's credentials and provide the requested consent. 
+
+3. Select the checkboxes and click **Share** to share the resources with Oliver and Pam.
+
+    ![photo-editor-share](../assets/img/learn/photo-editor-share.png)
+
+    !!! info
+        Once the resources are shared, a XACML policy is created to configure the access control policy in the authorization server. 
+        
+        To view the policy, return to the [management console](https://localhost:9443/carbon) and click **Entitlement > PAP > Policy > album-1-policy**.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-uma.md]: Management Console

Comment (YvonneW) [reply en/docs/learn/configuring-access-delegation-with-uma.md]: Under the **Main** menu, click ...

File: en/docs/learn/configuring-access-delegation-with-uma.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+3. Select the checkboxes and click **Share** to share the resources with Oliver and Pam.
+
+    ![photo-editor-share](../assets/img/learn/photo-editor-share.png)
+
+    !!! info
+        Once the resources are shared, a XACML policy is created to configure the access control policy in the authorization server. 
+        
+        To view the policy, return to the [management console](https://localhost:9443/carbon) and click **Entitlement > PAP > Policy > album-1-policy**. 
+
+4. Navigate to the following URL in an incognito browser window to access the **Photo Viewer** application: <http://localhost.com:8080/photo-view/>.
```
Comment (YvonneW) [inline en/docs/learn/configuring-access-delegation-with-uma.md]: Isn't incognito Chrome-specific?

File: en/docs/learn/configuring-single-sign-on-wsfed.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ This tutorial demonstrates configuring WS-Federation for one application and ret
     wso2server.bat run
     ```
     
-3. Log in using admin/admin credentials and create a new user called "Alex" with login permission. For instructions, see [Adding Users and Roles](../../learn/adding-users-and-roles#create-a-user).
+3. Log in to the management console using admin/admin credentials and create a new user called "Alex" with login permission. For instructions, see [Adding Users and Roles](../../learn/adding-users-and-roles#create-a-user).
```
Comment (YvonneW) [inline en/docs/learn/configuring-single-sign-on-wsfed.md]: Management Console

File: en/docs/learn/deploying-the-sample-app.md [hunk 7]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+2. [Download](https://github.com/wso2/samples-is/releases/download/v4.1.0/photo-edit.war) the `photo-edit.war` file from the
+   latest release assets.
+3. [Download](https://github.com/wso2/samples-is/releases/download/v4.1.0/photo-view.war) the `photo-view.war` file from the
+   latest release assets.
+
+### Deploy the sample web app
+
+Deploy the sample web apps on a web container.
+
+1.  Copy the .war files into the `           webapps          `
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]:
``` tab="Example"
```

File: en/docs/learn/deploying-the-sample-app.md [hunk 8]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+    folder. For example,
+    `           <TOMCAT_HOME>/apache-tomcat-<version>/webapps          `
+    .
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]:
``` tab="Example"
<TOMCAT_HOME>/apache-tomcat-<version>/webapps 
```

File: en/docs/learn/deploying-the-sample-app.md [hunk 9]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+2.  Start the Tomcat server.
+
+To check the **photo-edit** sample application, navigate to
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]: 3. To check ...

File: en/docs/learn/deploying-the-sample-app.md [hunk 10]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+`          http://<TOMCAT_HOST>:<TOMCAT_PORT>/photo-edit/index.jsp         `
+on your browser.
+
+For example,
+`                     http://localhost.com:8080/photo-edit/index.jsp                 .         `
+
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]:
```tab="Example"
http://localhost.com:8080/photo-edit/index.jsp
```

File: en/docs/learn/deploying-the-sample-app.md [hunk 11]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+To check the **photo-edit** sample application, navigate to
+`          http://<TOMCAT_HOST>:<TOMCAT_PORT>/photo-edit/index.jsp         `
+on your browser.
+
+For example,
+`                     http://localhost.com:8080/photo-edit/index.jsp                 .         `
+    
+### Configuring the service provider
+
+Once the application is deployed, the relevant service providers are created on the management console using dynamic client registration. Follow the steps below to view the service providers and the configurations.
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]: Management Console

File: en/docs/learn/deploying-the-sample-app.md [hunk 12]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+
+1.  Return to the WSO2 IS management console.
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]: Management Console

File: en/docs/learn/deploying-the-sample-app.md [hunk 13]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+
+2.  Navigate to **Main**>**Identity**>**Service Providers** and click **List**. Note that the `photo-editor` and `photo-view` service providers have been created using dynamic client registration. 
+
+    ![photo-samples-service-providers](../assets/img/learn/photo-samples-service-providers.png)
+
+3.  Click **Edit** to view the photo-editor and photo-view service provider configurations.
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]: To view the photo-editor and photo-view service provider configurations, click **Edit**.

File: en/docs/learn/deploying-the-sample-app.md [hunk 14]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+
+1.  Return to the WSO2 IS management console.
+
+2.  Navigate to **Main**>**Identity**>**Service Providers** and click **List**. Note that the `photo-editor` and `photo-view` service providers have been created using dynamic client registration. 
+
+    ![photo-samples-service-providers](../assets/img/learn/photo-samples-service-providers.png)
+
+3.  Click **Edit** to view the photo-editor and photo-view service provider configurations. 
+
+3.  Expand **Inbound Authentication Configuration** section and then expand **OAuth/OpenID Connect Configuration**.
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]: Under the **Inbound Authentication Configuration** section, click **OAuth/OpenID Connect Configuration**.

File: en/docs/learn/deploying-the-sample-app.md [hunk 15]
```diff
@@ -594,7 +594,66 @@ on your browser.
...
+
+    ![photo-samples-service-providers](../assets/img/learn/photo-samples-service-providers.png)
+
+3.  Click **Edit** to view the photo-editor and photo-view service provider configurations. 
+
+3.  Expand **Inbound Authentication Configuration** section and then expand **OAuth/OpenID Connect Configuration**. 
+
+    ![oauth-configs-for-photo-app](../assets/img/learn/oauth-configs-for-photo-app.png)
+
+4. Click **Edit** to view the OAuth/OpenID Connect configurations.
```
Comment (YvonneW) [inline en/docs/learn/deploying-the-sample-app.md]: To view the ...., .....

File: en/mkdocs.yml [hunk 16]
```diff
@@ -86,6 +86,7 @@ nav:
           - 'WS-Federation (Passive STS)': learn/configuring-single-sign-on-wsfed.md
         - 'Access Delegation':
           - 'OAuth 2.0': learn/configuring-access-delegation-with-oauth2.md
+          - 'User Managed Access (UMA)': 'learn/configuring-access-delegation-with-uma.md'
```
Comment (YvonneW) [inline en/mkdocs.yml]: UMA

File: en/docs/learn/deploying-the-sample-app.md (hunk 8 above)
Comment (sherenewso2) [reply en/docs/learn/deploying-the-sample-app.md]: this is already in monospace

File: en/docs/learn/deploying-the-sample-app.md (hunk 10 above)
Comment (sherenewso2) [reply en/docs/learn/deploying-the-sample-app.md]: this is already in monospace

**Pull request merged**